build_tools/cmake/build_android.sh

```diff
@@ -47,8 +47,11 @@ cd build-host
   -DCMAKE_INSTALL_PREFIX=./install \
   -DIREE_BUILD_COMPILER=ON \
   -DIREE_BUILD_TESTS=OFF \
+  -DIREE_BUILD_BENCHMARKS=ON \
   -DIREE_BUILD_SAMPLES=OFF
 "${CMAKE_BIN?}" --build . --target install
+# Also make sure that we can generate artifacts for benchmarking on Android.
+"${CMAKE_BIN?}" --build . --target iree-benchmark-suites
```

Comment:
How much does this add to execution time?

Reply:
From `cmake_android_arm64-v8a`:

```
[[22:50:27 UTC] /usr/bin/cmake --build . --target iree-benchmark-suites
[0/2] Re-checking globbed directories...
[1/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV3Small-fp32,imagenet/iree-dylib__CPU-ARM64-v8A__3-thread,big-core,full-inference/flagfile
[2/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV2-fp32,imagenet/iree-dylib__CPU-ARM64-v8A__3-thread,big-core,full-inference/flagfile
[3/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV2-fp32,imagenet/iree-vulkan__GPU-Mali-Valhall__kernel-execution/flagfile
[4/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV3Small-fp32,imagenet/iree-dylib-sync__CPU-ARM64-v8A__1-thread,big-core,full-inference/flagfile
[5/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV3Small-fp32,imagenet/iree-vulkan__GPU-Mali-Valhall__full-inference/flagfile
[6/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV3Small-fp32,imagenet/iree-vulkan__GPU-Adreno__full-inference/flagfile
[7/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV2-fp32,imagenet/iree-vulkan__GPU-Adreno__full-inference/flagfile
[8/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV2-fp32,imagenet/iree-vulkan__GPU-Mali-Valhall__full-inference/flagfile
[9/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV2-fp32,imagenet/iree-dylib-sync__CPU-ARM64-v8A__1-thread,big-core,full-inference/flagfile
[10/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV3Small-fp32,imagenet/iree-vulkan__GPU-Mali-Valhall__kernel-execution/flagfile
[11/22] Downloading https://storage.googleapis.com/iree-model-artifacts/MobileNetV3Small-b0c5c584.tar.gz
[12/22] Downloading https://storage.googleapis.com/iree-model-artifacts/MobileNetV2-b0c5c584.tar.gz
[13/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV2-fp32,imagenet/iree-vulkan__GPU-Mali-Valhall__full-inference/compiled.vmfb
[14/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV2-fp32,imagenet/iree-vulkan__GPU-Adreno__full-inference/compiled.vmfb
[15/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV3Small-fp32,imagenet/iree-vulkan__GPU-Adreno__full-inference/compiled.vmfb
[16/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV3Small-fp32,imagenet/iree-vulkan__GPU-Mali-Valhall__full-inference/compiled.vmfb
[17/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV2-fp32,imagenet/iree-vulkan__GPU-Mali-Valhall__kernel-execution/compiled.vmfb
[18/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV3Small-fp32,imagenet/iree-vulkan__GPU-Mali-Valhall__kernel-execution/compiled.vmfb
[19/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV2-fp32,imagenet/iree-dylib-sync__CPU-ARM64-v8A__1-thread,big-core,full-inference/compiled.vmfb
[20/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV2-fp32,imagenet/iree-dylib__CPU-ARM64-v8A__3-thread,big-core,full-inference/compiled.vmfb
[21/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV3Small-fp32,imagenet/iree-dylib-sync__CPU-ARM64-v8A__1-thread,big-core,full-inference/compiled.vmfb
[22/22] Generating /tmpfs/src/github/iree/build-host/benchmark_suites/TensorFlow/MobileNetV3Small-fp32,imagenet/iree-dylib__CPU-ARM64-v8A__3-thread,big-core,full-inference/compiled.vmfb
[[22:50:42 UTC] cd /tmpfs/src/github/iree
```

So it's adding (42 - 27 =) 15 seconds for now. It's downloading time + compilation time. I'd suspect downloading will take the majority eventually given this only have two models; we will increase and have more models in the future. But seems a minor addition to the time we already have anyway.